src/main/resources/assets/curios/lang/pt_br.json

```diff
@@ -2,36 +2,43 @@
   "curios.name": "Curios",
   "curios.slot": "Slot",
   "curios.cosmetic": "Cosmético",
-  "commands.curios.set.success": "Tipo %s agora foi definido como %d slots para %s",
-  "commands.curios.add.success": "%d slots do tipo %s foram adicionados para %s",
+  "commands.curios.set.success": "O tipo %s agora foi definido para %d slots para %s",
+  "commands.curios.add.success": "%d slots do tipo %s foram adicionados a %s",
   "commands.curios.remove.success": "%d slots do tipo %s foram removidos de %s",
-  "commands.curios.unlock.success": "O slot %s foi desbloqueado para %s",
-  "commands.curios.lock.success": "O slot %s foi bloqueado por %s",
-  "commands.curios.clear.success": "O slot %s foi limpo por %s ",
-  "commands.curios.clearAll.success": "Todos os slots de %s foram limpos",
+  "commands.curios.unlock.success": "Slot %s foi desbloqueado por %s",
+  "commands.curios.lock.success": "Slot %s foi bloqueado por %s",
+  "commands.curios.clear.success": "Slot %s foi limpo para %s",
+  "commands.curios.clearAll.success": "Todos os slots foram limpo para %s",
   "commands.curios.reset.success": "Redefinir slots para %s",
-  "argument.curios.type.unknown": "Tipo de slot de curio desconhecido %s",
-  "curios.identifier.curio": "Raridade",
+  "argument.entity.options.curios.description": "Inventário do curios da entidade",
+  "argument.curios.type.unknown": "Tipo de slot de Curio desconhecido %s",
+  "curios.identifier.curio": "Curio",
```

Comment:
I'm not sure what the correct translation should be here, but this slot is supposed to be "curio" as in "accessory" or something else suitably generic. It's meant to be a very non-specific type of slot. It's not just the name of the mod. It works in English as-is but it should probably be translated in order to get across the actual meaning. But I don't know if "Curio" also happens to work in Portuguese, so just let me know one way or the other.